ecommerce/backend/API/views/vendor.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import exceptions
+from rest_framework.response import Response
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework import status
+import json
+
+from ..utils import permissions, Role
+from ..models import Vendor,ImageUrls
+
+@api_view(['GET'])
+@permission_classes([permissions.AllowAnonymous]) 
+def vendor_details(request,vendor_id):
+    #print(vendor_id)
+    vendor = Vendor.objects.filter(id=vendor_id).first()
```

Comment:
please check whether that vendor really exists and return 400_BAD_REQUEST if it does not.
```suggestion
    vendor = Vendor.objects.filter(id=vendor_id).first()
    if vendor is None:
        return Response(status=status.HTTP_400_BAD_REQUEST)
```

Reply:
don't forget to delete debug comments also